### DISCUSSION TITLE
Any ways to integrate or make BPM (Business Process Management) workflow system  with budibase app

### DISCUSSION THREAD

**Discussion by cwmine** [Budibase/budibase#3377] [Ideas]
As the title, want to use budibase to make a workflow app with muti-user filling and reviewing form app.

Is it able to use budibase to make such system or integrate other BPM system in some way?

**Comment by shogunpurple** [comment]
Hi @cwmine - it would be good to know more about what exactly you are trying to do. Are you trying to integrate with this BPM system using an HTTP API or similar? 

You can create approval apps and form filling apps with budibase very easily. Guides for form flows are here: https://docs.budibase.com/building-apps/design/forms

If we understand some more about your BPM system, and how you intend to integrate with it, we can provide some more solutions.

Thanks!
